pkg/client/typed/discovery/discovery_client.go

```diff
@@ -343,3 +344,12 @@ func stringDoesntExistIn(str string, slice []string) bool {
 	}
 	return true
 }
+
+// GetRESTClient returns a RESTClient that is used to communicate
```

Comment:
I guess I would rather make the method RESTClient() and change all the variables to be named restClient. Maybe that can be a separate change, since it'll be big and need lots of rebases and just a rename.